js/main.js

```diff
@@ -0,0 +1,109 @@
+
+        next = document.getElementById('next');
+        counter = 0;
+        prev = document.getElementById('prev');
+        items = document.getElementsByTagName('img');
+        amount = items.length;
+        current = document.getElementsByClassName('current');
+
+        function navigate(direction){
+            if(direction === 1){
+                items = document.getElementById('img'+counter);
+                current = document.getElementsByClassName('current');
+                removeCurrentAddActive(items);
+            }
+        }
+        next.addEventListener("click", function(event){
+            items = document.getElementById('img'+counter);
+            counter = counter+1;
+
+            if(counter <= amount-1){
+                removeCurrentAddActive(items);
+                removeActiveAddCurrent(items.nextSibling);
+            }else if(counter === amount){
+                removeCurrentAddActive(items);
+                items = document.getElementById('img0');
+                removeActiveAddCurrent(items);
+                counter=0;
+            }
+            navigate(0);
+        });
+        prev.addEventListener("click", function(event){
+            items = document.getElementById('img'+counter);
+
+            if(counter > 0){
+                counter = counter-1;
+                removeCurrentAddActive(items);
+                removeActiveAddCurrent(items.previousSibling);
+            }else if(counter === 0){
+                items = document.getElementById('img'+counter);
+                removeCurrentAddActive(items);
+                counter = amount-1;
+                items = document.getElementById('img'+counter);
+                removeActiveAddCurrent(items);
+            }
+            navigate(-1);
+        });
+    };
+    function photoData(pics, myResp){
+        for(var i=0; i< myResp.photos.photo.length; i++){
+            farm = myResp.photos.photo[i].farm;
+            server = myResp.photos.photo[i].server;
+            id = myResp.photos.photo[i].id;
+            title = myResp.photos.photo[i].title;
+            secret = myResp.photos.photo[i].secret;
+            picUrl = "https://farm"+farm+".staticflickr.com/"+server+"/"+id+"_"+secret+"_z.jpg";
+            pics.push(picUrl);
+        };
+    }
+    function pupulate(pics){
+        for(var i=0; i<pics.length; i++){
+            list = document.getElementById('list');
+            img = '';
+            for(var i = 0; i<pics.length; i++){
+                list.innerHTML = list.innerHTML + '<li class='+'active'+ ' id=img'+[i]+'><img src=' + pics[i] + ' /></li>';
+            }
+        }
+    }
+    function removeActiveAddCurrent(items){
+        items.classList.remove('active');
+        items.classList.add('current');
+    };
+    function removeCurrentAddActive(items){
+        items.classList.remove('current');
+        items.classList.add('active');
+    }
+
+    window.onload = function execute(){
```

Comment:
this can simply be `window.onload = loadPics`, since we can assign a reference to our function as an event handler